src/main/java/uk/ac/ebi/ega/egafuse/service/EgaDirectory.java

```diff
@@ -75,17 +75,15 @@ public EgaPath find(String path) {
 
     @Override
     public void getattr(FileStat stat) {
-        stat.st_mode.set(FileStat.S_IFDIR | 0777);
-        stat.st_uid.set(0);
-        stat.st_gid.set(0);
+        stat.st_mode.set(FileStat.S_IFDIR | 0644);
```

Comment:
Correct me if I'm wrong should it not be 444 (r--r--r--) ?

Reply:
Done